cmd/agent/commands/root.go

```diff
@@ -55,6 +56,11 @@ func BuildRootCmd(env runtime.Environment, subcommands []*cobra.Command) *RootCo
 
 // Do executes the RootCommand
 func (r *RootCommand) Do(ctx context.Context) error {
+	sc := r.env.Signal().Notify(os.Interrupt)
```

Comment:
I would also suscribe to `SIGHUP` and `SIGTERM` here. While I'm not 100% sure of on which cases the disruptor will see these, I think it's a sane default for being very common in the unix world.